dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_EMU.rst

```diff
@@ -0,0 +1,106 @@
+.. _Muon_Analysis_EMU-ref:
+
+Muon Unscripted Testing: EMU
+============================
+
+.. contents:: Table of Contents
+   :local:
+
+Introduction
+------------
+
+These tests are for the EMU instrument at ISIS.
+This section will show some advance fitting and plotting features of the GUI.
+It would be worth restarting Mantid before doing the second test.
```

Comment:
Can we drop this line?
A user won't restart Mantid so if it causes crashes or problems we'd rather look into / fix these instead

Reply:
This is just to clear the ADS. To make it easier to find the results table etc. Since we are not giving them useful names to save time

Reply:
Could we ask users to clear the ADS then?
Clearing the ADS without closing Mantid for extended periods of time is (unfortunately) how a lot of users use their data reduction machines

Reply:
```suggestion
It would be worth closing the muon GUI and clearing the ADS before doing the second test.
This is to give a clean state so it is easier to find the correct results table.
```